src/main/java/edu/ucsb/cs156/dining/repositories/DiningCommonsRepository.java

```diff
@@ -0,0 +1,15 @@
+package edu.ucsb.cs156.dining.repositories;
```

Comment:
Why is this here? What end user functionality does it support?

There is no issue among the five approved issues that describes creating an table for a DiningCommons (unless it is an oversight, in which case please point it out.)

This should be removed.